### PR TITLE
Docs for machine health checks

### PR DESCRIPTION
**Pull request by mitalipaygude**
*Issue #, if available:*

*Description of changes:*
Docs for machine health checks

*Documentation added/planned (if applicable):*
Added
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/getting-started/optional/healthchecks.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+### __nodeStartupTimeout__ (optional)
+* __Description__: determines how long a Machine Health Check should wait for a Node to join the cluster, before considering a Machine unhealthy.
+* __Default__: ```10m0s```
+* __Minimum Value (If configured)__: ```30s```
+* __Type__: string
+
+### __unhealthyMachineTimeout__ (optional)
+* __Description__: if the unhealthy condition is matched for the duration of this timeout, the Machine is considered unhealthy.
+* __Default__: ```5m0s```
```
Comment (abhinavmpandey08) [inline docs/content/en/docs/getting-started/optional/healthchecks.md:42]: Does this also have a min value?

File: docs/content/en/docs/getting-started/optional/healthchecks.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    unhealthyMachineTimeout: "5m0s"
+```
+## Machine Health Check Spec Details
+### __machineHealthCheck__ (optional)
+* __Description__: top level key; required to configure machine health check timeouts.
+* __Type__: object
+
+### __nodeStartupTimeout__ (optional)
+* __Description__: determines how long a Machine Health Check should wait for a Node to join the cluster, before considering a Machine unhealthy.
+* __Default__: ```10m0s```
```
Comment (abhinavmpandey08) [inline docs/content/en/docs/getting-started/optional/healthchecks.md:36]: Let's mention that the default is 20m for tinkerbell

File: docs/content/en/docs/getting-started/optional/healthchecks.md (hunk 1 above)
Comment (mitalipaygude) [reply docs/content/en/docs/getting-started/optional/healthchecks.md:42]: no, this does not have a minimum value.

**Review by abhinavmpandey08 — APPROVED** [review]

**Pull request merged**